README.md

```diff
@@ -54,8 +54,8 @@ jobs:
 | git-commit-message | Commit message | `terraform-docs: automated action` | false |
 | git-push | If true it will commit and push the changes | `false` | false |
 | git-push-sign-off | If true it will sign-off commit | `false` | false |
-| git-push-user-email | If empty the no-reply email of the PR author will be used (i.e. `${GITHUB\_ACTOR}@users.noreply.github.com`) | `""` | false |
-| git-push-user-name | If empty the name of the PR author will be used (i.e. `${GITHUB\_ACTOR}`) | `""` | false |
+| git-push-user-email | If empty the no-reply email of the GitHub Actions bot will be used (i.e. `github-actions[bot]@users.noreply.github.com`) | `""` | false |
+| git-push-user-name | If empty the name of the GitHub Actions bot will be used (i.e. `github-actions[bot]`) | `""` | false |
```

Comment:
This table is auto-generated using the content of `action.yml`. Please update the action file and run `./scripts/update-readme.sh` (which in this case shouldn't change README any further)

Reply:
Done, didn't know about that